test/api/models/outcomes/ResponseWrapperSpec.scala

```diff
@@ -14,17 +14,17 @@
  * limitations under the License.
  */
 
-package v1.models.outcomes
+package api.models.outcomes
 
 import support.UnitSpec
 
 class ResponseWrapperSpec extends UnitSpec {
 
-  "map" should {
-    "apply the partial function to the responseData" in {
-      val partialFunction: Int => String = _.toString
+  "mapping a ResponseWrapper" should {
+    "return the same response wrapper with correlationId and responseData" in {
+      val response = ResponseWrapper("X-123", "Response")
 
-      ResponseWrapper("", 1).map(partialFunction) shouldBe ResponseWrapper("", "1")
+      response.map(a => a) shouldBe ResponseWrapper("X-123", "Response")
```

Comment:
Hmm... I guess this is copied from somewhere (looks like similar in other repos). Would be better if it didn't just use the identity.